app/views/schools/core_programme/materials/success.html.erb

```diff
@@ -1,5 +1,27 @@
+<% academic_year = Cohort.active_registration_cohort.description %>
+
 <% content_for :title, "Core induction materials selected" %>
 
-<%= govuk_panel title_text: "Study materials saved", text: nil, classes: "govuk-!-margin-bottom-8" %>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        
+        <%= govuk_panel title_text: "Training materials confirmed", classes: "govuk-!-margin-bottom-7" do %>
+            Academic year
+            <br/>
+            <strong><%= academic_year %></strong>
```

Comment:
```suggestion
            <strong><%= @cohort.description %></strong>
```

Can you try this instead of:

```ruby
<strong><%= academic_year %></strong>
```

and

```ruby
<% academic_year = Cohort.active_registration_cohort.description %>
```

Reply:
Added... second part was already there tho at the top. Doesnt look any different to what i had.
`<% academic_year = Cohort.active_registration_cohort.description %>`

Reply:
Thanks! It's not different but is more dynamic in the sense that we might want to use that template for previous registration cohorts.